### PR TITLE
Enable the kiwi container for 16.0

### PR DESCRIPTION
**Pull request by dirkmueller**
We need it to deliver SL Micro 6.0 updates

### CODE REVIEW

File: src/bci_build/package/versions.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def get_pkg_version(pkg_name: str, os_version: OsVersion) -> str:
 #: projects from which to take package versions
 _OBS_PROJECTS: dict[OsVersion, str] = {
     **{OsVersion(ver): f"SUSE:SLE-15-SP{ver}:Update" for ver in range(3, 8)},
+    **{OsVersion.SLE16_0: "SUSE:SLFO:Main:Build"},
     **{OsVersion.TUMBLEWEED: "openSUSE:Factory"},
```
Comment (dcermak) [inline src/bci_build/package/versions.py:130]: Nit: these two could be one dictionary:
```suggestion
    **{OsVersion.SLE16_0: "SUSE:SLFO:Main:Build", OsVersion.TUMBLEWEED: "openSUSE:Factory"},
```

**Review by dcermak — COMMENTED** [review]
Sorry, github mobile didn't submit my comment from Friday… LGTM, only have a question about the SLE 16 project name

File: src/bci_build/package/versions.py [hunk 2]
```diff
@@ -126,6 +126,7 @@ def get_pkg_version(pkg_name: str, os_version: OsVersion) -> str:
 #: projects from which to take package versions
 _OBS_PROJECTS: dict[OsVersion, str] = {
     **{OsVersion(ver): f"SUSE:SLE-15-SP{ver}:Update" for ver in range(3, 8)},
+    **{OsVersion.SLE16_0: "SUSE:SLFO:Main:Build"},
```
[the remark this answers is not in the document]
Comment (dirkmueller) [reply src/bci_build/package/versions.py:129]: prior maintenance phase, the sles16 product project only includes the release packages. everything else comes from main. so for now (and at some point we need to fix that, but it's a year out) :main is the one that carries the packages. 

unless we switch to the component architecture model of course.

File: src/bci_build/package/versions.py (hunk 1 above)
Comment (dirkmueller) [reply src/bci_build/package/versions.py]: resolved

File: src/bci_build/package/versions.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dcermak) [reply src/bci_build/package/versions.py]: > unless we switch to the component architecture model of course.

Let's not assume the worst case scenario :grimacing:

File: src/bci_build/package/versions.py [hunk 3]
```diff
@@ -125,9 +125,9 @@ def get_pkg_version(pkg_name: str, os_version: OsVersion) -> str:
 
 #: projects from which to take package versions
 _OBS_PROJECTS: dict[OsVersion, str] = {
-    **{OsVersion(ver): f"SUSE:SLE-15-SP{ver}:Update" for ver in range(3, 8)},
-    **{OsVersion.TUMBLEWEED: "openSUSE:Factory"},
-}
+    OsVersion.SLE_16_0: "SUSE:SLFO:Main:Build",
```
[the remark this answers is not in the document]
Comment (dirkmueller) [reply src/bci_build/package/versions.py]: more was broken.. fixed now.

**Pull request merged**